src/rest/model/mod.rs

```diff
@@ -4,6 +4,7 @@ mod futures;
 mod markets;
 mod orders;
 mod positions;
+mod spotmargin;
```

Comment:
rename to `spot_margin` to be consistent with the FTX docs?

Reply:
Thanks, I was on the fence about doing that.  Done